src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -104,12 +106,14 @@
           :style               {:border-radius 16
                                 :opacity       (if (and outgoing (= outgoing-status :sending)) 0.5 1)}
           :on-press            (fn []
-                                 (when (and outgoing
-                                            (not= outgoing-status :sending)
-                                            (not @show-delivery-state?))
-                                   (reset! show-delivery-state? true)
-                                   (js/setTimeout #(reset! show-delivery-state? false)
-                                                  delivery-state-showing-time-ms)))
+                                 (if (and platform/ios? @keyboard-shown)
```

Comment:
@flexsurfer because on Android we hide KB using the back button. On, iOS KB is hidden by tapping on outside the composer. `keyboard-shown` flag is necessary to know whether to hide the keyboard or execute the other function.